packages/migrate/bin/migrate.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+const yargs = require('yargs');
+const { run } = require('jscodeshift/src/Runner');
+const { join, resolve } = require('path');
+
+const cli = yargs
+  .scriptName('migrate')
+  .command(
+    '$0 [files..]',
+    'Migrate Neutrino middleware to the latest version via codemods',
+    (yargs) => {
+      yargs.positional('files', {
+        array: true,
+        description: 'files to migrate',
+        default: ['.neutrinorc.js']
+      });
+    }
+  )
+  .option('dry', {
+    alias: 'd',
+    description: 'dry run (no changes are made to files)',
+    boolean: true
+  })
+  .option('print', {
+    alias: 'p',
+    description: 'print transformed files to stdout, useful for development',
+    boolean: true
+  })
+  .option('silent', {
+    alias: 's',
+    description: 'do not write to stdout or stderr',
+    boolean: true
+  })
+  .help()
+  .wrap(null)
+  .argv;
+
+run(
+  resolve(__dirname, '../transforms/middleware.js'),
+  cli.files.map(file => join(process.cwd(), file)),
```

Comment:
If I pass an invalid filename, I get:

```
migrate invalid
Skipping path C:\Users\Ed\src\treeherder\invalid which does not exist.
TypeError: array is not iterable
    at concatAll (C:\Users\Ed\src\neutrino\node_modules\jscodeshift\src\Runner.js:72:27)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

Should we try and avoid the `TypeError` here, or is it not worth it?